example/setting_example.json

```diff
@@ -1,7 +1,8 @@
 {
   "client_id": "<your client id here>",
   "client_secret": "<your client secret here>",
+  "port": 3001,
   "url":  {
-    "redirect": "http://localhost/connect/res"
+    "redirect": "http://localhost:3001/connect/res"
```

Comment:
OK, how about this instead?

- Change the port to 3001
- Fix the `url.redirect` setting to include `:3001` and do not make it a configurable option

The problem with using `process.argv` to configure the port is that the `url.redirect` setting will be broken if you change to a different port.